.travis.yml

```diff
@@ -1,19 +1,19 @@
 # This file is part of Requirements-Builder
-# Copyright (C) 2015, 2016, 2017, 2018 CERN.
+# Copyright (C) 2015, 2016, 2017, 2018, 2019 CERN.
 #
 # Requirements-Builder is free software; you can redistribute it and/or
 # modify it under the terms of the Revised BSD License; see LICENSE
 # file for more details.
 
-sudo: false
+dist: xenial
+sudo: true
```

Comment:
__dist: xenial__ is no longer required and could be detrimental in the future and Travis CI has completely deprecated the __sudo__ tag.

Reply:
okay, nice.